resources/js/manager.js

```diff
@@ -19,9 +19,13 @@ import MetricasComponent from './components/MetricasComponent';
 import DepartamentosComponent from './components/DepartamentosComponent';
 import ConfiguracionComponent from './components/ConfiguracionComponent';
 
+import ConvocatoriasCreateComponent from './components/convocatorias/ConvocatoriasCreateComponent';
+
 const routes = [
-    { path: '/', component: DashboardComponent },
+    { path: '/', redirect: '/dashboard' },
```

Comment:
Remove the redirect put component again, the route base is '/manager/' so '/manager/dashboard' breaks the route path design

Reply:
listo